tests/integration/cri-containerd/integration-tests.sh

```diff
@@ -612,6 +612,12 @@ function main() {
 	# Make sure the right artifacts are going to be built
 	sudo make clean
 
+	# the latest containerd had an issue for its e2e test, thus we should do the following
+	# fix to workaround this issue. For much info about this issue, please see:
+	# https://github.com/containerd/containerd/pull/11240
+	# Once this pr was merged and release new version, we can remove this workaround.
+	sed -i 's/cat "\${config_file}"/cat "\${CONTAINERD_CONFIG_FILE}"/' script/test/utils.sh
```

Comment:
Is this a fix for containerd 2.0 only ? What about containerd 1.7 ?

Reply:
Actually this fix isn't only for specific version, instead it would work for all of the versions.